_posts/2017-07-21-pydelhi-conf-2017-a-beautiful-conference-happend-at-new-delhi-india.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: post
+title: "PyDelhi Conf 2017: A beautiful conference happened in New Delhi, India"
+date: "2017-07-21 18:14:03 +0530"
+tag:
+  - python
+  - conference
+  - django
+  - talks
+---
+
+![PyDelhi Conf
+2017]({{site.url}}/assets/images/pydelhi_conf_2017/group_photo.jpg)
+
+## TL;DR
+
+[PyDelhi conf 2017][pydelhi_conf_2017] was a two-day conference which featured
+workshops, a dev sprints, both full-length and lightning talks. If you are a
+beginner, this conference is worth attending because there are free workshops.
+If you are an experienced developer, then you will surely get advanced level
+talks for you. Delhiites should not miss the chance to attend this conference. I
+conducted a workshop titled **“Tango with Django”** helping beginners to
+understand the Django web framework.
+
+
+## Detailed Review
+
+
+### About the [PyDelhi community][pydelhi_community]
+
+
+![PyDelhi
+Community]({{site.url}}/assets/images/pydelhi_conf_2017/pydelhi_community.jpg)
+
+<p class="center"> PyDelhi conf 2017 volunteers </p>
+
+The [PyDelhi community][pydelhi_community] was known as NCR Python Users Group
+before few years. This community is performing a role of an umbrella
+organization for other FLOSS communities across New Delhi, India. They are
+actively arranging monthly [meetups][pydelhi_meetup] on interesting topics. Last
+[PyCon India](https://in.pycon.org/2016/) which is a national level conference
+of Python programming language was impressively organized by this community.
+This year too they took the responsibility of managing it. I am very thankful to
+this community for their immense contribution to this society. If you are around
+New Delhi, India then you should not miss the chance to attend their
+[meetups][pydelhi_meetup]. This community has great people who are always happy
+to mentor.
+
+
+### [PyDelhi conf 2017][pydelhi_conf_2017]
+
+
+![Conference T-shirt]({{site.url}}/assets/images/pydelhi_conf_2017/t_shirt.jpg)
+
+<p class="center"> Conference T-shirt </p>
+
+PyDelhi conf is a regional level conference of Python programming language
+organized by PyDelhi community. It is their second year organizing this
+conference. Last year it was located at [JNU University](http://www.jnu.ac.in).  This
+year it happened at [IIM, Lucknow](https://www.iiml.ac.in/) campus based in
+Noida, New Delhi, India. I enjoyed various talks which I will mention later
+here, a workshops section because I was conducting one and some the panel
+discussions because people involved were having a good level of experience. 80%
+of the time slot was divided equally between 30 minutes talk and 2-hour workshop
+section.  10% were given to panel discussions and 10% was preserved for
+lightning talks.  The dev sprints were happening parallel with the conference.
+The early slot was given to workshops for both the days. One large conference
+hall was located on a 2nd floor of the building and two halls were at a ground
```

Comment:
`the ground floor`, assuming there was only 1 ground floor. If many, then it should be `were at ground floor X`